src/EditorFeatures/Core/Shared/Utilities/ThreadingContext+DenyExecutionSynchronizationContext.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.ExceptionServices;
+using System.Threading;
+
+namespace Microsoft.CodeAnalysis.Editor.Shared.Utilities
+{
+    internal sealed partial class ThreadingContext
+    {
+        private sealed class DenyExecutionSynchronizationContext : SynchronizationContext
```

Comment:
Not sure what all this is about, or if we need it. Can we justify it before I review it? :smile:

Reply:
⌛️ Fixed locally and will be in next commit.